py/desispec/quicklook/qlconfig.py

```diff
@@ -172,32 +189,39 @@ def qaargs(self):
         
         for PA in self.palist:
             for qa in self.qalist[PA]: #- individual QA for that PA
+                
                 params=self._qaparams(qa)
-                qaopts[qa]={'camera': self.camera, 'paname': PA, 'PSFFile': '%%PSFFile', 'amps': self.amps, 'qafile': self.dump_qa()[0][0][qa],'qafig': self.dump_qa()[0][1][qa], 'FiberMap': '%%FiberMap', 'param': params, 'qlf': self.qlf}
+                qaopts[qa]={'camera': self.camera, 'paname': PA, 'PSFFile': self.psf, 'amps': self.amps, 'qafile': self.dump_qa()[0][0][qa],'qafig': self.dump_qa()[0][1][qa], 'FiberMap': self.fibermap, 'param': params, 'qlf': self.qlf}
                 
         return qaopts 
    
     def _qaparams(self,qa):
-     
+            
         params={}
-        if qa == 'Count_Pixels':
-            params[qa]= dict(
-                             CUTLO = 100,
-                             CUTHI = 500
-                             )
-        elif qa == 'CountSpectralBins':
-            params[qa]= dict(
-                             CUTLO = 100,   # low threshold for number of counts
-                             CUTMED = 250,
-                             CUTHI = 500
-                             )
-        elif qa == 'Sky_Residual':
-            params[qa]= dict(
-                             PCHI_RESID=0.05, # P(Chi^2) limit for bad skyfiber model residuals
-                             PER_RESID=95.,   # Percentile for residual distribution
-                             BIN_SZ=0.1,) # Bin size for residual/sigma histogram
+        if self.algorithms is not None:
+            for PA in self.palist:
+                if qa in self.qalist[PA]:
+                    params[qa]=self.algorithms[PA]['QA'][qa]['PARAMS']
+
         else:
-            params[qa]= None
+            if qa == 'Count_Pixels':
```

Comment:
Are these different from the offline?

Either way, I feel these should be written a file (e.g. desispec_params.yaml)
and read from disk.   Such parameter files can be archived and far more
easily modified than searching for and changing these parameters in this code.